src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -529,6 +531,9 @@ var _ = Describe("installer HostRoleMaster role", func() {
 					fmt.Errorf("bad-conditions-status")).Times(1)
 				setConsoleAsAvailable("cluster-id")
 
+				// Patching NS
+				mockk8sclient.EXPECT().PatchNamespace(defaultTestControllerConf.Namespace, aiNamespaceRunlevelPatch).Return(nil)
```

Comment:
Counter proposal: What do you think about explicitly creating the namespace in the e`BeforeEach` phase? This will (should?) guarantee that *every* test in the installer uses the namespace with that label set. Removal will only happen at the end of the flow, so I think that should be the exception in the test suite.


All that said, these are just unit-tests so I don't think setting the label will have any actual functional effect but it does get us closer to reality, at least in form.

Reply:
Given that I have already a PR for the next release that removes the label completely, I'm not sure if it's worth redesigning the tests in here